chapter07/pxd/ex5.pl

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+use feature "say";
+
+say "Enter a couple of lines (Ctrl-D to exit):";
+
+while (<>) {
+    
+    # Will match any double letters
+    if (/(\w)\1/) { 
```

Comment:
There's also `\S` to match any non-whitespace characters, so you could match 00 etc